### PR TITLE
WIP: Guide viz expr

### PR DESCRIPTION
**Pull request by davidmanzanares**
https://github.com/CartoDB/carto-vl/issues/969

@makella this is just a stub, but I prefer to get some feedback before continuing. Feel free to add new stuff or talk to me directly.

### CODE REVIEW

**Comment by makella** [comment]
ok! taking a look now

**Comment by makella** [comment]
@davidmanzanares ok, looked through, here are a couple of comments:

- in the outline, I don't see an item related to `width` so we should add that
- `strokeWidth` is inside of color, we probably want to take that out from color 
- what I don't see is what is outlined more in the reference which is using attributes and expressions. - also the expression types in the reference: numbers, categories, colors aren't outlined here, but I'm guessing that is covered in "What is an expression" (?)
- another thing that could be useful is to know about any caveats (for example, `order` is only for points that are sized by a numeric attribute...)

I know we have a guide for data-driven styling (https://github.com/CartoDB/carto-vl/issues/970) but the topics covered there are more about `ramp`.

In terms of a map-guided example, what are you thinking of using?

**Comment by makella** [comment]
@davidmanzanares sorry! i didn't actually read your guide... I only referred to the outline linked. :(

Reading now!

File: examples/guides/viz-properties/geom-types.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    map.addControl(nav, 'top-left');
+
+    // Define user
+    carto.setDefaultAuth({
+      user: 'cartovl',
+      apiKey: 'default_public'
+    });
+
+    // Define layer
+    const source = new carto.source.Dataset('ne_10m_populated_places_simple');
```
Comment (elenatorro) [inline examples/guides/viz-properties/geom-types.html]: use `populated_places` instead

File: examples/guides/viz-properties/accessing-viz-properties.html [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    map.addControl(nav, 'top-left');
+
+    // Define user
+    carto.setDefaultAuth({
+      user: 'cartovl',
+      apiKey: 'default_public'
+    });
+
+    // Define layer
+    const source = new carto.source.Dataset('ne_10m_populated_places_simple');
```
Comment (elenatorro) [inline examples/guides/viz-properties/accessing-viz-properties.html]: use `populated_places` instead

**Comment by davidmanzanares** [comment]
Merged into https://github.com/CartoDB/carto-vl/pull/1004

**Pull request closed without merging**